src/main/lrsql/admin/interceptors/lrs_management.clj

```diff
@@ -0,0 +1,30 @@
+(ns lrsql.admin.interceptors.lrs-management
+  (:require [clojure.spec.alpha :as s]
+            [io.pedestal.interceptor :refer [interceptor]]
+            [io.pedestal.interceptor.chain :as chain]
+            [lrsql.admin.protocol :as adp]))
+
+
+(def validate-delete-actor-params
+  (interceptor {:name ::validate-delete-actor-params
```

Comment:
It would be nice to start the `{:name ...` part on a new line to shorten line lengths and keep them under 80 columns.

Also did you forget to remove the extra spaces?